src/axes.jl

```diff
@@ -155,6 +155,14 @@ end
     return IdOffsetRange(r.parent[s .- r.offset], r.offset)
 end
 
+# offset-preserve broadcasting
+Base.broadcasted(::Base.Broadcast.DefaultArrayStyle{1}, ::typeof(-), r::IdOffsetRange{T}, x::Integer) where T =
```

Comment:
```suggestion
Broadcast.broadcasted(::Base.Broadcast.DefaultArrayStyle{1}, ::typeof(-), r::IdOffsetRange{T}, x::Integer) where T =
```
and similarly below.